runtime_builders/template_builder.py

```diff
@@ -98,7 +98,9 @@ def _resolve_tags(config_file):
                     arg = args[i]
                     m = re.search(IMAGE_REGEX, arg)
```

Comment:
Can you get capture group directly instead of needing to do substring manipulation?